.devcontainer/base/devcontainer.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Autoware",
+  "build": {
+    "dockerfile": "Dockerfile"
+  },
+  "remoteUser": "autoware",
+  "hostRequirements": {
+    "gpu": true
+  },
```

Comment:
This can be removed for this non-GPU file. Although it is not critical because if I remember correctly it just issues a warning in the logs, but doesn't block anything.

Reply:
resolving following https://github.com/autowarefoundation/autoware/pull/4072#discussion_r1514751869

Reply:
I'm not good at VSCode with Containers, I don't know if this affects anything related to RViz. If you think we can remove it safely, I can remove it.

Reply:
It won't change how things get setup. From https://containers.dev/implementors/json_reference/#min-host-reqs: "you will be presented with a warning if the requirements are not met". Remove this won't affect whether rviz works or not.

Reply:
@ambroise-arm I see, I will let @oguzkaganozt know this, let's remove it as well in the next PR.